onnxruntime/core/session/inference_session.cc

```diff
@@ -1394,7 +1398,7 @@ common::Status InferenceSession::ValidateInputs(const std::vector<std::string>&
       }
       auto expected_element_type = expected_type->AsTensorType()->GetElementType();
       auto input_element_type = input_ml_value.Get<Tensor>().DataType();
-      ORT_RETURN_IF_ERROR_SESSIONID_(CheckTypes(input_element_type, expected_element_type));
+      ORT_RETURN_IF_ERROR_SESSIONID_(CheckTypes(input_element_type, expected_element_type, "Tensor"));
```

Comment:
would it make sense to match the exact string representations used in onnx https://github.com/onnx/onnx/blob/master/onnx/defs/data_type_utils.cc#L121 ?
i.e.  tensor , sparse_tensor and seq 
rather than Tensor, SparseTensor and TensorSequence

Reply:
Makes sense. Fixed.